### PR TITLE
Add game scanner for Raze/Build engine games to JELOS

### PR DESCRIPTION
**Pull request by travis134**
## Description

Add game scanner for Raze/Build engine games to JELOS

This automates manual steps detailed here: https://github.com/JustEnoughLinuxOS/distribution/wiki/JELOS-Game-Engines-and-Associated-Launch-Files#build-files

Hopefully make life easier for users.

I also fixed some bugs with path vars in start_raze.sh.

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested Locally?

I've scp'd the script over to my device, ran it from `/storage/roms` and verified it emplaces the correctly formatted build files, some examples:

`/storage/roms/build/Blood/BLOOD.RFF`
`Blood.build`
```
PATH=Blood
GRP=BLOOD.RFF
-- end --

```

`/storage/roms/buils/Duke Nukem 3D Twentieth Anniversary World Tour/DUKE3D.GRP`
`Duke Nukem 3D Twentieth Anniversary World Tour.build`
```
PATH="Duke Nukem 3D Twentieth Anniversary World Tour"
GRP=DUKE3D.GRP
-- end --

```

**Test Configuration**:
* Build OS name and version: JELOS
* Docker (Y/N): Built with docker and ran on my RG353P -- works great
* JELOS Branch: dev

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/games/emulators/raze/sources/start_raze.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [ ${EXT} == "build" ]; then
     done < "${1}"
 fi
 
-if [[ ! "RUN_DIR" == "/storage/roms/build" ]]; then
+if [[ ! "$RUN_DIR" == "/storage/roms/build" ]]; then
```
Comment (travis134) [inline packages/games/emulators/raze/sources/start_raze.sh:43]: This is an unrelated fix -- shellcheck uncovered this dead comparison bug.

**Comment by travis134** [comment]
I'm fairly confident in these changes, but would like to build an image and run on my device before saying it's safe to merge. That might take awhile however.

**Comment by travis134** [comment]
Here it is in action https://user-images.githubusercontent.com/412428/185535222-94f80daf-6c44-448a-a409-a365b3edb91e.MOV

**Comment by travis134** [comment]
Long discussion in emulation-setup, but the per-game sections for keybinds is needed in the raze.ini file otherwise the raze engine writes default values which are keyboard based and don't work on joypads at all. These probably aren't the best defaults for most games, but it's at least better than not having working binds at all.

**Comment by romadu** [comment]
I have some defaults for each game so I’ll add any additional default
mappings that might be useful
